crates/router/src/core/payments/operations/payment_status.rs

```
@@ -462,16 +462,37 @@ async fn get_tracker_for_sync<
         };
 
     let merchant_id = payment_intent.merchant_id.clone();
-    let authentication = payment_attempt.authentication_id.clone().async_map(|authentication_id| async move {
+
+    let authentication_store = if let Some(ref authentication_id) =
+        payment_attempt.authentication_id
+    {
+        let authentication =
             db.find_authentication_by_merchant_id_authentication_id(
                     &merchant_id,
                     authentication_id.clone(),
                 )
                 .await
                 .to_not_found_response(errors::ApiErrorResponse::InternalServerError)
-                .attach_printable_lazy(|| format!("Error while fetching authentication record with authentication_id {authentication_id}"))
-        }).await
-        .transpose()?;
+                .attach_printable_lazy(|| format!("Error while fetching authentication record with authentication_id {authentication_id}"))?;
+
+        let tokenized_data = crate::core::payment_methods::vault::get_tokenized_data(
+            state,
+            &authentication.authentication_id,
+            false,
+            merchant_context.get_merchant_key_store().key.get_inner(),
+        )
+        .await
+        .ok(); // marking .ok() as token might be invalidated
```

Should we atleast log this? genuine errors may be ignored?